packages/permissions/src/components/authorized/authorized.js

```diff
@@ -1,66 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { wrapDisplayName } from 'recompose';
-import { defaultMemoize } from 'reselect';
-import warning from 'warning';
-import camelCase from 'lodash.camelcase';
-import upperFirst from 'lodash.upperfirst';
 import { ApplicationContext } from '@commercetools-frontend/application-shell-connectors';
 import {
   hasSomePermissions,
   hasEveryPermissions,
+  getUnconfiguredPermissions,
```

Comment:
The name unconfigured comes from the fact that there is no opposite to actual (from `actualPermissions`).

Reply:
How about simply `getInvalidPermissions`?